sky/adaptors/azure.py

```diff
@@ -2,9 +2,12 @@
 
 # pylint: disable=import-outside-toplevel
 import functools
+import json
+import subprocess
 import threading
 
 from sky.adaptors import common
+from sky.utils.subprocess_utils import run
```

Comment:
```suggestion
from sky.utils import subprocess_utils
```
nit: import package and modules only; https://google.github.io/styleguide/pyguide.html#2144-decision